Draft/references.bib

```diff
@@ -0,0 +1,66 @@
+@article{talby2006agile,
+  title={Agile software testing in a large-scale project},
+  author={Talby, David and Keren, Arie and Hazzan, Orit and Dubinsky, Yael},
+  journal={IEEE software},
+  volume={23},
+  number={4},
+  pages={30--37},
+  year={2006},
+  publisher={IEEE}
+}
+
+@inproceedings{cruzes2016communication,
+  title={Communication between developers and testers in distributed continuous agile testing},
+  author={Cruzes, Daniela S and Moe, Nils B and Dyb{\aa}, Tore},
+  booktitle={Global Software Engineering (ICGSE), 2016 IEEE 11th International Conference on},
```

Comment:
I suggest reviewing and amending the `booktitle` entries for conference proceedings to be more consistent.